### PR TITLE
Removed invalid errors, added more tests

### PR DESCRIPTION
**Pull request by kftang**

### CODE REVIEW

**Comment by kftang** [comment]
Fixes #82

**Comment by kftang** [comment]
Fixes #81

File: routes/foodtrucks_test.go [hunk 1]
```diff
@@ -517,6 +563,20 @@ func TestClaimFoodTruckPutUnauthorized(t *testing.T) {
 	}
 }
 
+func TestClaimFoodTruckPutNoFoodTruck(t *testing.T) {
+	tests.ClearDB()
+
+	req, _ := http.NewRequest("PUT", "/foodtruck", nil)
```
Comment (janinebar) [inline routes/foodtrucks_test.go]: wrong endpoint

File: routes/reviews_test.go [hunk 2]
```diff
@@ -411,3 +428,22 @@ func TestGetReviewWithID(t *testing.T) {
...
 }
+
+func TestReviewGetEmpty(t *testing.T) {
+	tests.ClearDB()
+
+	// Add sample review to DB
+	tests.AddReview(models.JSONReview{
+		ID: "test",
+	})
+	req, _ := http.NewRequest("GET", "/review", nil)
```
Comment (janinebar) [inline routes/reviews_test.go]: wrong endpoint. GET /reviews

File: routes/users_test.go [hunk 3]
```diff
@@ -389,3 +427,37 @@ func TestValidUpdateUserRequest(t *testing.T) {
...
 
 }
+
+func TestProfilePutInvalidBody(t *testing.T) {
+	tests.ClearDB()
+
+	updateProfileRequest := invalidRequestBody{}
+	body, _ := json.Marshal(updateProfileRequest)
+
+	req, _ := http.NewRequest("PUT", "/users", bytes.NewBuffer(body))
```
Comment (janinebar) [inline routes/users_test.go]: change endpoint "/profile"

**Review by janinebar — APPROVED** [review]

**Pull request merged**